dockerfiles/Dockerfile

```diff
@@ -177,8 +177,7 @@ RUN find /dpkg/ -type d -empty -delete && \
 # We want latest at time of build
 # hadolint ignore=DL3006
 FROM gcr.io/distroless/cc-debian12 AS production
-ARG RELEASE_VERSION
-ENV FLUENT_BIT_VERSION=${RELEASE_VERSION}
+EXPOSE 2020
```

Comment:
Moved up, as this is not likely to change, meaning the layer doesn't have to be rebuild,pushed,pulled.